src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
 
         /// <summary>
-        /// The optional breakpoint objects to use for debugging.
+        /// Gets or sets a flag that prevents PowerShell from automatically performing a BreakAll when the debugger is attached to the remote target.
         /// </summary>
-        [Experimental("Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints", ExperimentAction.Show)]
-        [Parameter(Position = 1,
-                   ParameterSetName = DebugRunspaceCommand.InstanceIdParameterSet)]
-        [Parameter(ParameterSetName = DebugRunspaceCommand.RunspaceParameterSet)]
-        [Parameter(ParameterSetName = DebugRunspaceCommand.IdParameterSet)]
-        [Parameter(ParameterSetName = DebugRunspaceCommand.NameParameterSet)]
-        public Breakpoint[] Breakpoint
-        {
-            get;
-            set;
-        }
+        [Experimental("Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace", ExperimentAction.Show)]
+        [Parameter]
+        public SwitchParameter NoInitialBreak { get; set; }
```

Comment:
I want to go with that parameter name because it matches the intent and is consistent with `Enable-RunspaceDebug`; however, I took this approach to avoid changing the current behavior. Since it's an ad-hoc command though, I think the change is acceptable, especially once the PR that makes `Debug-Runspace` and `Debug-Job` consistent are implemented and merged in. This will be updated in the next commit.